### PR TITLE
Adding test coverage metric with eslint refinements

### PR DESCRIPTION
**Pull request by Tosch110**
Integrating Istanbul coverage metric.

Updating package.json dependencies.

As mentioned in Issue #10

Refining rules defined in `eslint_ecma5.json`.

### CODE REVIEW

File: eslint_ecma5.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "jsx": false
+    }
+  },
+  "rules": {
+    // Inspired by airbnb rules: https://github.com/airbnb/javascript/blob/master/linters/SublimeLinter/SublimeLinter.sublime-settings
+    // Force all variable names to use either camelCase style or UPPER_CASE with underscores.
+    "camelcase": [2, {"properties": "always"}],
+
+    // Prohibit use of == and != in favor of === and !==.
+    "eqeqeq": 2,
```
Comment (MaciejBaj) [inline eslint_ecma5.json]: in a case of consistency, I think it is better to keep one convention in actions naming, (there is 
`"eqeqeq": 2` and after that `"indent": ["error", "tab"]`). I think that the second notation is more readable, so `"eqeqeq": "error"`

File: eslint_ecma5.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    // Suppress warnings about == null comparisons.
+    "no-eq-null": 1,
+
+    // Enforce tab width of 2 spaces.
```
Comment (MaciejBaj) [inline eslint_ecma5.json]: description: enforce tab width of 2 spaces. -> tabs

File: package.json [hunk 3]
```diff
@@ -20,6 +21,14 @@
     "js-nacl": "LiskHQ/js-nacl#6dc1417"
   },
   "devDependencies": {
+    "eslint": "=3.14.1",
```
Comment (MaciejBaj) [inline package.json]: unused if we are using grunt-eslint

File: eslint_ecma5.json (hunk 2 above)
Comment (4miners) [inline eslint_ecma5.json]: I like 4 spaces ;)

**Pull request merged**